starknet-rpc-test/get_transaction_receipt.rs

```diff
@@ -156,18 +152,18 @@ async fn work_with_declare_transaction(madara: &ThreadSafeMadaraClient) -> Resul
 
     let fee_token_address = FieldElement::from_hex_be(FEE_TOKEN_ADDRESS).unwrap();
     let expected_fee =
-        FieldElement::from_hex_be("0x000000000000000000000000000000000000000000000000000000000000d3ae").unwrap();
+        FieldElement::from_hex_be("0x00000000000000000000000000000000000000000000000000000000000030fc").unwrap();
 
     assert_declare_tx_receipt(
         declare_tx_receipt,
         DeclareTransactionReceipt {
             transaction_hash: FieldElement::from_hex_be(
```

Comment:
Depending on the test execution the tx hash will change due to nonce being incremented